for-developers/walkthrough/dev-setup/making-your-first-changes.md

```diff
@@ -47,4 +47,214 @@ Now you should see some custom content in the post that you added to the feed. Y
 
 ## Make Your First Core Change
 
-**WIP**
+Your first core change will consist of adding the support of emoji reactions to posts.
+We will implement this change by creating a new transaction from scratch.
+
+### Step 1: Development
+
+First, navigate to the `network.go` file in the `lib` folder.
+The `network.go` file defines all the data structures used to form both messages and transactions over the network.
+The first set of changes consists of adding constants to identify the new transaction type you are creating.
+
+In `network.go`, perform the following changes:
+* Add a new `TxnType` constant named `TxnTypeReact` and update the value of `NEXT_ID` to `27`.
```

Comment:
Do we want to swap out to `27` and instead say increment NEXT_ID to the next integer?

Reply:
Good call, thanks!